jupyterlab_automl/src/service/model.ts

```diff
@@ -53,12 +66,35 @@ export interface ModelEvaluation {
   confusionMatrix: any[];
 }
 
+function formatTime(data: Model | Pipeline) {
+  data.createTime = new Date(data.createTime);
+  data.updateTime = new Date(data.updateTime);
+  if ('elapsedTime' in data) {
+    const humanizer = humanizeDuration.humanizer({
+      round: true,
+      language: 'shortEn',
+      largest: 2,
+      languages: {
+        shortEn: {
+          y: () => 'y',
+          mo: () => 'mo',
+          w: () => 'w',
+          d: () => 'day',
+          h: () => 'hr',
+          m: () => 'min',
+          s: () => 'sec',
+        },
+      },
+    });
+    data.elapsedTime = humanizer(Math.round(data.elapsedTime as number) * 1000);
+  }
+}
+
 export abstract class ModelService {
```

Comment:
Just out of curiosity, what's the intent of the this being an abstract class with only static methods? Could you accomplish the same thing with a module containing only functions?